sgd/sgd-aa-oas3.yml

```
@@ -0,0 +1,577 @@
+      content:
+        application/problem+json:
+          schema:
+            $ref: "#/components/schemas/Errors"
+          example:
+            error: "invalid_request"
+            error_description:
+              The request is malformed, a required parameter is missing or a
+              parameter has an invalid value.
+    "401":
+      description:
+        The client request has not been completed because it lacks valid
+        authorization credentials for the requested resource.
+      content:
+        application/problem+json:
+          schema:
+            $ref: "#/components/schemas/Errors"
+          example:
+            error: "invalid_client"
+            error_description:
+              The client request has not been completed because it lacks valid
+              authorization credentials for the requested resource.
+    "404":
+      description: The requested resource was not found
+      content:
+        application/problem+json:
+          schema:
+            $ref: "#/components/schemas/Errors"
+          example:
+            error: "not_found"
+            error_description: The requested resource was not found
+    "429":
+      description: Too Many Requests
+      content:
+        application/problem+json:
+          schema:
+            $ref: "#/components/schemas/Errors"
+          example:
+            error: "too_many_requests"
+            error_description:
+              The rate limit was exceeded and access is temporarily blocked. It
+              is allowed max 50 requests per hour.
+    "500":
+      description: Internal server error.
+      content:
+        application/problem+json:
+          schema:
+            $ref: "#/components/schemas/Errors"
+          example:
+            error: "temporarily_unavailable"
+            error_description:
+              The server is currently unable to handle the request due to
+              maintenance or temporary overload.
+  schemas:
+    Authorization:
+      type: string
+      example: DPoP Kz~8mXK1EalYznwH-LC-1fBAo.4Ljp~zsPE_NeO.gxU
+    DPoP:
+      type: string
+      example:
+        "eyJ0eXAiOiJkcG9wK2p3dCIsImFsZyI6IkVTMjU2IiwiandrIjp7Imt0eSI6IkVDIiwieCI6Imw4dEZyaHgtMzR0VjNoUklDUkRZOXpDa0RscEJoRjQyVVFVZldWQVdCRnMiLCJ5IjoiOVZFNGpmX09rX282NHpiVFRsY3VOSmFqSG10NnY5VERWclUwQ2R2R1JEQSIsImNydiI6IlAtMjU2In19.eyJqdGkiOiJlMWozVl9iS2ljOC1MQUVCIiwiaHRtIj      oiR0VUIiwiaHR1IjoiaHR0cHM6Ly9yZXNvdXJjZS5leGFtcGxlLm9yZy9wcm90ZWN0Z      WRyZXNvdXJjZSIsImlhdCI6MTU2MjI2MjYxOCwiYXRoIjoiZlVIeU8ycjJaM0RaNTNFc05yV0JiMHhXWG9hTnk1OUlpS0NBcWtzbVFFbyJ9.2oW9RP35yRqzhrtNP86L-Ey71E    OptxRimPPToA1plemAgR6pxHF8y6-yqyVnmcw6Fy1dqd-jfxSYoMxhAJpLjA"
+    DelegationList:
+      type: object
+      description:
+        "JSON Object containing the delegation list. Each delegation is
+        identified by the parameter id and the related fiscal number of
+        delegating is given partially hidden for privacy reason"
+      properties:
+        delegation_list:
```

```suggestion
        items:
```
Lo schema indica già di cosa stiamo parlando.